packages/envd/internal/host/mmds.go

```diff
@@ -138,6 +140,7 @@ func PollForMMDSOpts(ctx context.Context, mmdsChan chan<- *MMDSOpts, envVars *ut
 
 			envVars.Store("E2B_SANDBOX_ID", mmdsOpts.InstanceID)
 			envVars.Store("E2B_TEMPLATE_ID", mmdsOpts.EnvID)
+			envVars.Store("E2B_EVENTS_ADDRESS", mmdsOpts.EventProxyIP)
```

Comment:
It's called `E2B_EVENTS_ADDRESS` when you are putting IPv4 here. Let's set the full HTTP endpoint address here, including the `http://` prefix.

In the future, when the `address` received from MMDS is by Hyperloop One, we will use it here.